### PR TITLE
Add rudimentary clock

### PR DESCRIPTION
**Pull request by jackthorley**
Work In Progress.

### CODE REVIEW

File: src/Clock.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        currentdate: now
+    });
+  }
+
+  componentWillMount() {
+    this.setTime();
+  }
+
+  componentDidMount() {
+    window.setInterval(function () {this.setTime();}.bind(this), 1000);
```
Comment (Codesleuth) [inline src/Clock.jsx]: Why not just...
```js
window.setInterval(this.setTime.bind(this), 1000)
```

Comment (jackthorley) [reply src/Clock.jsx]: Yes!

**Comment by Codesleuth** [comment]
Cheers, @jackthorley. I think it still looks like a turd, but at least you took it with me :)

**Comment by jackthorley** [comment]
Haha, not even a turd rolled in glitter. I might spend some time working on
it for knowledge gain and i guess for people who integrate with hackbot

**Pull request merged**